packages/mdc-dialog/foundation.js

```diff
@@ -127,6 +131,8 @@ class MDCDialogFoundation extends MDCFoundation {
     this.adapter_.removeClass(cssClasses.OPEN);
     this.adapter_.removeBodyClass(cssClasses.SCROLL_LOCK);
 
+    this.adapter_.setAttribute('aria-hidden', 'true');
```

Comment:
Screen readers usually don't pronounce elements that are set to "display: none". Do we need this?

Reply:
OK, so now I'm confused...

I've seen the following quote several places:

> Authors MUST set aria-hidden="true" on content that is not displayed, regardless of the mechanism used to hide it.

However, the [current WAI-ARIA spec](https://www.w3.org/TR/using-aria/#fourth) apparently contradicts this:

> If an interactive element is hidden using display:none or visibility:hidden, it will also be removed from the accessibility tree, which makes the addition of aria-hidden="true" unnecessary.

So yeah... in that case, maybe any other components we have that are currently doing this don't need to do it? I'll have to double-check tomorrow.